scylla-cdc-base/src/main/java/com/scylladb/cdc/cql/CQLConfiguration.java

```diff
@@ -25,6 +76,38 @@ private CQLConfiguration(List<InetSocketAddress> contactPoints,
         // or provided user-password pair.
         Preconditions.checkArgument((this.user == null && this.password == null)
                 || (this.user != null && this.password != null));
+
+        this.consistencyLevel = Preconditions.checkNotNull(consistencyLevel);
+        this.localDCName = localDCName;
```

Comment:
localDCName can be null?

Reply:
Yes, if it was not configured. In such case, `DCAwareRoundRobinPolicy` is not set as the load balancing policy.